locales/ru_RU/LC_MESSAGES/duckduckgo.po

```diff
@@ -3206,7 +3206,7 @@ msgid "Privacy Essentials"
 msgstr "Основы конфиденциальности"
 
 msgid "Privacy Essentials for %s"
-msgstr ""
+msgstr "Основы конфиденциальности для %s"
```

Comment:
I think we can fix this one with Google translate

Reply:
Good catch. I will update.